### DISCUSSION TITLE
Access Supabase logs via JDBC (or other)

### DISCUSSION THREAD

**Discussion by mnethercott** [supabase/supabase#18178] [Questions] [answered]
Hi all, 

We love the Supabase project but it would be great to access e.g: Edge or realtime logs externally via JDBC (or similar). This way we can export the logs to a central location for analysis. 

Does anyone know if this is possible? I can't find a way to access the Supabase logs available in the 'logs explorer' via JDBC. 

Thanks in advance. 
Max

**Comment by Ziinc** [comment, marked as answer]
API for accessing Supabase Logs is not available yet, please stay tuned as it is something currently being worked on.

**Reply by Ziinc** [reply]
as an addendum, direct JDBC is not something that we are looking to support at the moment.
